dll/win32/shell32/lang/fr-FR.rc

```diff
@@ -457,7 +457,7 @@ BEGIN
     CONTROL "", IDC_FILETYPES_ICON, "STATIC", SS_BITMAP | SS_REALSIZEIMAGE, 85, 140, 10, 10
     LTEXT "Nom de l'application", IDC_FILETYPES_APPNAME, 100, 140, 80, 10
     PUSHBUTTON "&Modifier...", IDC_FILETYPES_CHANGE, 180, 140, 50, 14, WS_TABSTOP
-    LTEXT "", IDC_FILETYPES_DESCRIPTION, 12, 155, 160, 30
+    LTEXT "", IDC_FILETYPES_DESCRIPTION, 12, 155, 160, 45
```

Comment:
The new height 45 for IDC_FILETYPES_DESCRIPTION is too big. It would overflow the surrounding groupbox in case more text than in french would get added. I will experiment with those RCs myself. Do not merge this PR meanwhile please.

Reply:
Assuming police height of 9px and 5 lines being needed, 45px looks like the minima.
Waiting for your inputs.